api/rest/v1/find_entities.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Find DCIDs
```

Comment:
The text is "Find DCIDs" while the API is /find/entities. This makes me wonder what is the difference when first reading it. Is there a concern of just using "Find Entities"?

Reply:
People usually ask us "how do I find a DCID?" so I thought that title would be more descriptive. You're right that the difference is confusing though. Updated the titles to "Find Entities".